spec/bolt/transport/ssh_spec.rb

```diff
@@ -106,7 +108,7 @@ def result_value(stdout = nil, stderr = nil, exit_code = 0)
       expect_node_error(Bolt::Node::ConnectError,
                         'CONNECT_ERROR',
                         /Failed to connect to/) do
-        ssh.with_connection(make_target(h: hostname, p: 65535)) {}
+        ssh.with_connection(make_target(port_: 65535)) {}
```

Comment:
Does this have a default host_ now?

Reply:
It always did. I can add that back, but it was unnecessary.

Reply:
No need, was just curious